posts/nightly-package-count.md

```diff
@@ -0,0 +1,6 @@
+# Stackage Nightly Package Count over Time
```

Comment:
This needs a metadata block at the top, such as this one:

```
---
author: Chris Dornan
title: Stackage nightly snapshots to switch to GHC 9.0.1
description: We will be releasing LST-18 and switching nightly to GHC 9
timestamp: 2021-05-29T16:07:00Z
---
```

Reply:
oops, fixed!